packages/babel-parser/test/fixtures/esprima/invalid-syntax/migrated_0222/output.json

```diff
@@ -13,8 +13,7 @@
     }
   },
   "errors": [
-    "SyntaxError: Octal literal in strict mode (1:36)",
-    "SyntaxError: Octal literal in strict mode (1:57)"
```

Comment:
Does anyone else see why this might have changed? There are two octal literals in this test, so it seems correct to report two errors.